lib/glific/flows/flow.ex

```diff
@@ -366,6 +366,8 @@ defmodule Glific.Flows.Flow do
 
   @spec validate_flow(map()) :: Keyword.t()
   defp validate_flow(flow) do
+    all_nodes = flow_objects(flow, :node)
```

Comment:
Let's move this to else statement. We don't need to fetch all the nodes for empty flow.

Reply:
@pankaj made requested changes